systemd/systemd.go

```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
 		"Resident memory size in bytes.",
 		[]string{"name"}, nil,
 	)
+
+	ipIngressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_ingress_bytes"),
+		"Service unit ingress IP accounting in bytes.",
+		[]string{"name"}, nil,
+	)
+	ipEgressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_egress_bytes"),
+		"Service unit egress IP accounting in bytes.",
+		[]string{"name"}, nil,
+	)
+	ipIngressPackets := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_ingress_packets"),
```

Comment:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_ingress_packets_total"),
```

as per https://prometheus.io/docs/practices/naming/

As per